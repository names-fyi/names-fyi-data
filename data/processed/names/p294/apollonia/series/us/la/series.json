[{"y":1985,"t":9,"m":0,"f":9,"mr":null,"fr":498,"mp":0,"fp":0.00029920212765957447},{"y":1986,"t":11,"m":0,"f":11,"mr":null,"fr":418,"mp":0,"fp":0.00039008475477853824}]