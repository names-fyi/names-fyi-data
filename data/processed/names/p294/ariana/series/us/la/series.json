[{"y":1984,"t":7,"m":0,"f":7,"mr":null,"fr":594,"mp":0,"fp":0.00022765708338753741},{"y":1987,"t":7,"m":0,"f":7,"mr":null,"fr":547,"mp":0,"fp":0.0002659877645628301},{"y":1989,"t":17,"m":0,"f":17,"mr":null,"fr":264,"mp":0,"fp":0.0006599891295908068},{"y":1990,"t":14,"m":0,"f":14,"mr":null,"fr":322,"mp":0,"fp":0.0005513981882630957},{"y":1991,"t":18,"m":0,"f":18,"mr":null,"fr":258,"mp":0,"fp":0.0007185055085422321},{"y":1992,"t":12,"m":0,"f":12,"mr":null,"fr":347,"mp":0,"fp":0.0005064573309698658},{"y":1993,"t":14,"m":0,"f":14,"mr":null,"fr":301,"mp":0,"fp":0.0006059032285986324},{"y":1994,"t":15,"m":0,"f":15,"mr":null,"fr":272,"mp":0,"fp":0.000669224591772999},{"y":1995,"t":16,"m":0,"f":16,"mr":null,"fr":276,"mp":0,"fp":0.0007282658170232135},{"y":1996,"t":16,"m":0,"f":16,"mr":null,"fr":272,"mp":0,"fp":0.0007424593967517402},{"y":1997,"t":16,"m":0,"f":16,"mr":null,"fr":270,"mp":0,"fp":0.0007394398742952213},{"y":1998,"t":14,"m":0,"f":14,"mr":null,"fr":320,"mp":0,"fp":0.000640790919077261},{"y":1999,"t":9,"m":0,"f":9,"mr":null,"fr":453,"mp":0,"fp":0.0004084596532631388},{"y":2000,"t":29,"m":0,"f":29,"mr":null,"fr":164,"mp":0,"fp":0.0013234757210660824},{"y":2001,"t":15,"m":0,"f":15,"mr":null,"fr":292,"mp":0,"fp":0.000722160704828848},{"y":2002,"t":29,"m":0,"f":29,"mr":null,"fr":163,"mp":0,"fp":0.0014133242360738828},{"y":2003,"t":36,"m":0,"f":36,"mr":null,"fr":130,"mp":0,"fp":0.0017336864916927522},{"y":2004,"t":32,"m":0,"f":32,"mr":null,"fr":147,"mp":0,"fp":0.001539349624783529},{"y":2005,"t":25,"m":0,"f":25,"mr":null,"fr":185,"mp":0,"fp":0.0012702606574869163},{"y":2006,"t":30,"m":0,"f":30,"mr":null,"fr":150,"mp":0,"fp":0.0015003000600120023},{"y":2007,"t":24,"m":0,"f":24,"mr":null,"fr":201,"mp":0,"fp":0.0011336261867649143},{"y":2008,"t":28,"m":0,"f":28,"mr":null,"fr":167,"mp":0,"fp":0.0013847675568743817},{"y":2009,"t":29,"m":0,"f":29,"mr":null,"fr":171,"mp":0,"fp":0.0014045624061607013},{"y":2010,"t":23,"m":0,"f":23,"mr":null,"fr":205,"mp":0,"fp":0.001168877369517711},{"y":2011,"t":19,"m":0,"f":19,"mr":null,"fr":249,"mp":0,"fp":0.0009732609363794693},{"y":2012,"t":27,"m":0,"f":27,"mr":null,"fr":168,"mp":0,"fp":0.0013711151736745886},{"y":2013,"t":37,"m":0,"f":37,"mr":null,"fr":127,"mp":0,"fp":0.0018375049662296384},{"y":2014,"t":46,"m":0,"f":46,"mr":null,"fr":111,"mp":0,"fp":0.0022014836085187846},{"y":2015,"t":42,"m":0,"f":42,"mr":null,"fr":116,"mp":0,"fp":0.0019757267852102737},{"y":2016,"t":28,"m":0,"f":28,"mr":null,"fr":198,"mp":0,"fp":0.0013364517206815904},{"y":2017,"t":28,"m":0,"f":28,"mr":null,"fr":183,"mp":0,"fp":0.0014100110786584752},{"y":2018,"t":34,"m":0,"f":34,"mr":null,"fr":136,"mp":0,"fp":0.0017386857581181284},{"y":2019,"t":27,"m":0,"f":27,"mr":null,"fr":175,"mp":0,"fp":0.0014237502636574562},{"y":2020,"t":27,"m":0,"f":27,"mr":null,"fr":178,"mp":0,"fp":0.001464684821525442},{"y":2021,"t":39,"m":0,"f":39,"mr":null,"fr":105,"mp":0,"fp":0.0021061727061619053},{"y":2022,"t":23,"m":0,"f":23,"mr":null,"fr":208,"mp":0,"fp":0.001300610721556209}]