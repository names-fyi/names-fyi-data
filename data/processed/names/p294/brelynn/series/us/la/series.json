[{"y":2015,"t":5,"m":0,"f":5,"mr":null,"fr":914,"mp":0,"fp":0.00023520556966788973},{"y":2016,"t":5,"m":0,"f":5,"mr":null,"fr":876,"mp":0,"fp":0.00023865209297885541}]