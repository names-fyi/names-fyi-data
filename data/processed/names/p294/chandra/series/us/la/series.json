[{"y":1956,"t":5,"m":0,"f":5,"mr":null,"fr":658,"mp":0,"fp":0.0001269454388503821},{"y":1957,"t":10,"m":0,"f":10,"mr":null,"fr":461,"mp":0,"fp":0.00025119316754584274},{"y":1960,"t":9,"m":0,"f":9,"mr":null,"fr":520,"mp":0,"fp":0.00023183925811437405},{"y":1961,"t":10,"m":0,"f":10,"mr":null,"fr":491,"mp":0,"fp":0.0002629157355067701},{"y":1964,"t":10,"m":0,"f":10,"mr":null,"fr":487,"mp":0,"fp":0.0002753910552985239},{"y":1965,"t":5,"m":0,"f":5,"mr":null,"fr":686,"mp":0,"fp":0.00014959758250306676},{"y":1966,"t":23,"m":0,"f":23,"mr":null,"fr":263,"mp":0,"fp":0.0007176063149355714},{"y":1967,"t":14,"m":0,"f":14,"mr":null,"fr":366,"mp":0,"fp":0.00044857417494392825},{"y":1968,"t":30,"m":0,"f":30,"mr":null,"fr":220,"mp":0,"fp":0.0009901970492127933},{"y":1969,"t":28,"m":0,"f":28,"mr":null,"fr":241,"mp":0,"fp":0.0009093566301841447},{"y":1970,"t":29,"m":0,"f":29,"mr":null,"fr":223,"mp":0,"fp":0.0009534769028439915},{"y":1971,"t":32,"m":0,"f":32,"mr":null,"fr":204,"mp":0,"fp":0.001098071511907213},{"y":1972,"t":25,"m":0,"f":25,"mr":null,"fr":230,"mp":0,"fp":0.0009424360085950164},{"y":1973,"t":25,"m":0,"f":25,"mr":null,"fr":220,"mp":0,"fp":0.0009732549538677152},{"y":1974,"t":27,"m":0,"f":27,"mr":null,"fr":208,"mp":0,"fp":0.0010874818753020784},{"y":1975,"t":28,"m":0,"f":28,"mr":null,"fr":200,"mp":0,"fp":0.0010954616588419405},{"y":1976,"t":25,"m":0,"f":25,"mr":null,"fr":220,"mp":0,"fp":0.0009607624610891203},{"y":1977,"t":23,"m":0,"f":23,"mr":null,"fr":253,"mp":0,"fp":0.0008218394911741585},{"y":1978,"t":18,"m":0,"f":18,"mr":null,"fr":298,"mp":0,"fp":0.0006541648495420846},{"y":1979,"t":14,"m":0,"f":14,"mr":null,"fr":369,"mp":0,"fp":0.0004784198475891057},{"y":1980,"t":11,"m":0,"f":11,"mr":null,"fr":456,"mp":0,"fp":0.0003582011788075157},{"y":1981,"t":18,"m":0,"f":18,"mr":null,"fr":303,"mp":0,"fp":0.0005878510777269758},{"y":1982,"t":10,"m":0,"f":10,"mr":null,"fr":475,"mp":0,"fp":0.0003156864602077217},{"y":1983,"t":8,"m":0,"f":8,"mr":null,"fr":540,"mp":0,"fp":0.00026001040041601667},{"y":1984,"t":7,"m":0,"f":7,"mr":null,"fr":600,"mp":0,"fp":0.00022765708338753741},{"y":1985,"t":11,"m":0,"f":11,"mr":null,"fr":430,"mp":0,"fp":0.0003656914893617021},{"y":1986,"t":16,"m":0,"f":16,"mr":null,"fr":308,"mp":0,"fp":0.0005673960069506011},{"y":1987,"t":8,"m":0,"f":8,"mr":null,"fr":503,"mp":0,"fp":0.00030398601664323444},{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":729,"mp":0,"fp":0.00019001292087861973},{"y":1990,"t":7,"m":0,"f":7,"mr":null,"fr":567,"mp":0,"fp":0.00027569909413154784}]