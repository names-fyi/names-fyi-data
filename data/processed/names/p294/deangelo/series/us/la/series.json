[{"y":1980,"t":6,"m":6,"f":0,"mr":597,"fr":null,"mp":0.00016411378555798687,"fp":0},{"y":1984,"t":9,"m":9,"f":0,"mr":451,"fr":null,"mp":0.0002508431115694417,"fp":0},{"y":1985,"t":7,"m":7,"f":0,"mr":526,"fr":null,"mp":0.00019526346620547295,"fp":0},{"y":1986,"t":6,"m":6,"f":0,"mr":580,"fr":null,"mp":0.0001762062788170685,"fp":0},{"y":1987,"t":8,"m":8,"f":0,"mr":479,"fr":null,"mp":0.0002487407499533611,"fp":0},{"y":1988,"t":11,"m":11,"f":0,"mr":385,"fr":null,"mp":0.00034835481521360484,"fp":0},{"y":1989,"t":10,"m":10,"f":0,"mr":421,"fr":null,"mp":0.000313361744798195,"fp":0},{"y":1990,"t":13,"m":13,"f":0,"mr":366,"fr":null,"mp":0.0004160399398342241,"fp":0},{"y":1991,"t":12,"m":12,"f":0,"mr":386,"fr":null,"mp":0.00038734667527437055,"fp":0},{"y":1992,"t":8,"m":8,"f":0,"mr":513,"fr":null,"mp":0.0002622692849883618,"fp":0},{"y":1993,"t":7,"m":7,"f":0,"mr":559,"fr":null,"mp":0.00023940627244433804,"fp":0},{"y":1994,"t":9,"m":9,"f":0,"mr":480,"fr":null,"mp":0.00032025050706330286,"fp":0},{"y":1995,"t":5,"m":5,"f":0,"mr":671,"fr":null,"mp":0.00018369521290275177,"fp":0},{"y":1996,"t":12,"m":12,"f":0,"mr":380,"fr":null,"mp":0.00044972454371697333,"fp":0},{"y":1998,"t":10,"m":10,"f":0,"mr":449,"fr":null,"mp":0.0003671880737313652,"fp":0},{"y":2000,"t":8,"m":8,"f":0,"mr":544,"fr":null,"mp":0.00029363185905670764,"fp":0},{"y":2001,"t":5,"m":5,"f":0,"mr":743,"fr":null,"mp":0.0001919975424314569,"fp":0},{"y":2002,"t":5,"m":5,"f":0,"mr":735,"fr":null,"mp":0.00019425019425019425,"fp":0},{"y":2004,"t":7,"m":7,"f":0,"mr":608,"fr":null,"mp":0.0002747468404113353,"fp":0},{"y":2005,"t":6,"m":6,"f":0,"mr":641,"fr":null,"mp":0.0002488077959776073,"fp":0},{"y":2009,"t":5,"m":5,"f":0,"mr":829,"fr":null,"mp":0.00019977625059932876,"fp":0},{"y":2010,"t":10,"m":10,"f":0,"mr":480,"fr":null,"mp":0.00041978003526152295,"fp":0},{"y":2011,"t":5,"m":5,"f":0,"mr":789,"fr":null,"mp":0.00021093486331420858,"fp":0},{"y":2017,"t":5,"m":5,"f":0,"mr":821,"fr":null,"mp":0.000209881207236704,"fp":0},{"y":2018,"t":9,"m":9,"f":0,"mr":524,"fr":null,"mp":0.00039473684210526315,"fp":0}]