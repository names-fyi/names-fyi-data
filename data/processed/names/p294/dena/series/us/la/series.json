[{"y":1955,"t":10,"m":0,"f":10,"mr":null,"fr":444,"mp":0,"fp":0.0002661556478228468},{"y":1956,"t":14,"m":0,"f":14,"mr":null,"fr":368,"mp":0,"fp":0.0003554472287810699},{"y":1957,"t":9,"m":0,"f":9,"mr":null,"fr":496,"mp":0,"fp":0.0002260738507912585},{"y":1958,"t":12,"m":0,"f":12,"mr":null,"fr":425,"mp":0,"fp":0.0003052813676605271},{"y":1959,"t":13,"m":0,"f":13,"mr":null,"fr":418,"mp":0,"fp":0.0003279102030520873},{"y":1960,"t":26,"m":0,"f":26,"mr":null,"fr":270,"mp":0,"fp":0.0006697578567748583},{"y":1961,"t":14,"m":0,"f":14,"mr":null,"fr":397,"mp":0,"fp":0.0003680820297094781},{"y":1962,"t":29,"m":0,"f":29,"mr":null,"fr":248,"mp":0,"fp":0.0007619948499658416},{"y":1963,"t":25,"m":0,"f":25,"mr":null,"fr":267,"mp":0,"fp":0.0006919266004262268},{"y":1964,"t":22,"m":0,"f":22,"mr":null,"fr":300,"mp":0,"fp":0.0006058603216567526},{"y":1965,"t":11,"m":0,"f":11,"mr":null,"fr":448,"mp":0,"fp":0.00032911468150674685},{"y":1966,"t":20,"m":0,"f":20,"mr":null,"fr":298,"mp":0,"fp":0.000624005491248323},{"y":1967,"t":19,"m":0,"f":19,"mr":null,"fr":296,"mp":0,"fp":0.0006087792374239026},{"y":1968,"t":20,"m":0,"f":20,"mr":null,"fr":291,"mp":0,"fp":0.0006601313661418623},{"y":1969,"t":29,"m":0,"f":29,"mr":null,"fr":232,"mp":0,"fp":0.0009418336526907213},{"y":1970,"t":27,"m":0,"f":27,"mr":null,"fr":237,"mp":0,"fp":0.0008877198750616472},{"y":1971,"t":27,"m":0,"f":27,"mr":null,"fr":230,"mp":0,"fp":0.0009264978381717109},{"y":1972,"t":24,"m":0,"f":24,"mr":null,"fr":239,"mp":0,"fp":0.0009047385682512157},{"y":1973,"t":26,"m":0,"f":26,"mr":null,"fr":216,"mp":0,"fp":0.0010121851520224238},{"y":1974,"t":19,"m":0,"f":19,"mr":null,"fr":280,"mp":0,"fp":0.0007652650233607218},{"y":1975,"t":16,"m":0,"f":16,"mr":null,"fr":316,"mp":0,"fp":0.0006259780907668231},{"y":1976,"t":11,"m":0,"f":11,"mr":null,"fr":409,"mp":0,"fp":0.00042273548287921294},{"y":1977,"t":9,"m":0,"f":9,"mr":null,"fr":513,"mp":0,"fp":0.00032158936611162723},{"y":1978,"t":8,"m":0,"f":8,"mr":null,"fr":542,"mp":0,"fp":0.00029073993312981537},{"y":1979,"t":13,"m":0,"f":13,"mr":null,"fr":397,"mp":0,"fp":0.000444247001332741},{"y":1980,"t":10,"m":0,"f":10,"mr":null,"fr":487,"mp":0,"fp":0.0003256374352795597},{"y":1981,"t":8,"m":0,"f":8,"mr":null,"fr":551,"mp":0,"fp":0.0002612671456564337},{"y":1982,"t":9,"m":0,"f":9,"mr":null,"fr":517,"mp":0,"fp":0.00028411781418694954},{"y":1983,"t":12,"m":0,"f":12,"mr":null,"fr":405,"mp":0,"fp":0.000390015600624025},{"y":1984,"t":8,"m":0,"f":8,"mr":null,"fr":551,"mp":0,"fp":0.0002601795238714713},{"y":1985,"t":7,"m":0,"f":7,"mr":null,"fr":626,"mp":0,"fp":0.00023271276595744682},{"y":1986,"t":6,"m":0,"f":6,"mr":null,"fr":661,"mp":0,"fp":0.0002127735026064754},{"y":1987,"t":6,"m":0,"f":6,"mr":null,"fr":637,"mp":0,"fp":0.0002279895124824258},{"y":1988,"t":6,"m":0,"f":6,"mr":null,"fr":638,"mp":0,"fp":0.0002280155050543437}]