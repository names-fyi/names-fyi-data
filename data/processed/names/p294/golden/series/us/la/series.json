[{"y":1918,"t":5,"m":0,"f":5,"mr":null,"fr":522,"mp":0,"fp":0.0002413942934389031},{"y":1919,"t":5,"m":0,"f":5,"mr":null,"fr":511,"mp":0,"fp":0.00022996964400699108},{"y":1920,"t":5,"m":5,"f":0,"mr":445,"fr":null,"mp":0.0002677376171352075,"fp":0},{"y":1924,"t":5,"m":0,"f":5,"mr":null,"fr":541,"mp":0,"fp":0.00020539785564638706},{"y":1929,"t":5,"m":5,"f":0,"mr":440,"fr":null,"mp":0.00026318559848405093,"fp":0}]